src/Symfony/Component/Serializer/Tests/Normalizer/AbstractNormalizerTest.php

```diff
@@ -103,4 +104,20 @@ public function testObjectToPopulateWithProxy()
 
         $this->assertSame('bar', $proxyDummy->getFoo());
     }
+
+    public function testObjectWithStaticConstructor()
+    {
+        $normalizer = $this->getMockBuilder(ObjectNormalizer::class)
```

Comment:
A mock allows to test the abstract class but i have no strong opinion about it and a fixture normalizer is fine for me too.